.jenkins_nightly

```diff
@@ -148,7 +148,7 @@ pipeline {
                         dockerfile {
                             filename 'Dockerfile.hipcc'
                             dir 'scripts/docker'
-                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-24.04:6.3.1-complete'
+                            additionalBuildArgs '--build-arg BASE=rocm/dev-ubuntu-24.04:6.3.1-complete --build-arg CMAKE_VERSION=3.31.3'
```

Comment:
is 3.31.3 the lowest that works here? Or did you just want to use the most recent anyways?

Reply:
Bruno went for the latest release.  As I understand it 3.28 was sufficient so I would have probably reached for 3.28.6 (latest patch release available) but I don't think it matters too much.